package.json

```diff
@@ -20,7 +20,8 @@
   "license": "ISC",
   "homepage": "https://github.com/leoasis/react-sound",
   "dependencies": {
-    "react": ">=0.13.3 <16.0.0",
+    "prop-types": "15.5.8",
```

Comment:
Should we do the same as in the [PropType's Readme](https://github.com/reactjs/prop-types#how-to-depend-on-this-package) and allow `prop-types` to start from 15.5.7? We definitely need to allow all possible prop-types versions, since this library doesn't do anything with them.